templates/user/register.html.twig

```diff
@@ -7,7 +7,11 @@
     {% else %}
         {{ form_start(form) }}
         {{ form_row(form.email) }}
-        {{ form_row(form.password) }}
+        <div data-role="check-password" data-route="{{ path('admin_user_ajax_password_strength') }}">
```

Comment:
hmm. Moeten we dat dan iedere keer zelf toevoegen?
Zou het niet logischer zijn om daar een FieldType van te maken? PasswordWithStrengthMetherType?

Reply:
Zal het in AC steken, maar zal niet voor direct zijn waarschijnlijk